packages/maskbook/src/social-network-adaptor/twitter.com/injection/ToolboxHint.tsx

```diff
@@ -48,8 +48,20 @@ const useStyles = makeStyles((theme) =>
     }),
 )
 
+function GetColor() {
+    const ele = document
+        .getElementsByTagName('header')[0]
+        .getElementsByTagName('nav')[0]
+        .getElementsByTagName('a')[1]
+        .getElementsByTagName('div')[0]
+        .getElementsByTagName('div')[1]
```

Comment:
Determine by the `...more` button, because it can't be selected but the other nav items can.

```suggestion
        .getElementsByTagName('header')[0]
        .getElementsByTagName('nav')[0]
        .getElementsByTagName('a')[1]
        .getElementsByTagName('div')[0]
        .getElementsByTagName('div')[1]
      	document.querySelector('header nav [role=button][aria-haspopup=menu] > div > div + div')
```

Reply:
We can also infer the current color from the background color 🤔 

![image](https://user-images.githubusercontent.com/1141198/114874158-a0e55380-9e2e-11eb-8b01-d8bc03d83c28.png)